content/en/docs/concepts/_index.md

```diff
@@ -1,17 +1,17 @@
 ---
 title: Concepts
 main_menu: true
-content_template: templates/concept
+# content_template: templates/concept
 weight: 40
 ---
 
-{{% capture overview %}}
+<!-- overview -->
```

Comment:
I like this strategy for replacement. 👍 

We do need to update [page templates](https://github.com/kubernetes/website/tree/master/layouts/partials/templates) and the [contribution docs](https://github.com/kubernetes/website/blob/master/content/en/docs/contribute/style/page-templates.md) accordingly.

Reply:
:+1:

Reply:
I moved `contribute/style/page-templates` to `contribute/style/page-styles` and cleaned up the template content.
There are many links to the `page-templates` page throughout the contribute section -- wow.
I did not deliberate on renaming the new page. Surely, the name and content can be changed.

Preview:
https://deploy-preview-21277--kubernetes-io-master-staging.netlify.app/docs/contribute/style/page-styles/

Reply:
Ideas to move forward:
With these changes, the site will be using (temporarily) two ways to render the pages.
One idea is to create single commits for migrating each localization or leave these changes as `en` only. I expect it would be easier to remove the templates and related code after the capture statements are removed.
Should decide whether to remove the content_template front matter or update as suggested.

Reply:
> single commits for migrating each localization

:+1: to that approach